src/components/shared/forms/tooltip.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
```

Comment:
It feels a bit odd to pass the main content (the stuff that causes a tooltip to appear) as a prop instead of as the children. This would feel more natural.

```jsx
<Tooltip content={getTooltip()}>
  <div>
    Some stuff that will show a tooltip if hovered over.
  </div>
</Tooltip>
```

Other react UI libs like Material and Bootstrap use this pattern too.

Reply:
Most of the react components I researched did it the way I have it actually. To me it made more sense for the contents of the Tooltip (which is the name of the component) to be the children of that component.  Also, the stuff inside a tooltip (the way our designers have it) is often more complicated than the thing that triggers it.

Reply:
Material UI does as you say. However, bootstrap does not. In that case the tooltip is just the toolip and they have an OverlayTrigger component where the tooltip is a prop.

Reply:
For the sake of getting it done and since its all likely to change in Vanadium, I'd say let's leave it as is. If you feel strongly otherwise, let's talk in person.

Reply:
The material tooltip is named differently but has the same behavior as the bootstrap version with overlay. So the factoring is a bit different but the implementation (and end usage) is essentially the same. The material design is also more opinionated about what a tooltip is so that allows it to get away with less customization than bootstrap. I think the tooltip content as a prop vs children is the more logical approach overall and given that that is also the design choice of those major libraries I think their years of experience in the wild testifies to that.